AdyenActions/UI/View Controllers/QR Code/QRCodeView.swift

```diff
@@ -0,0 +1,238 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
+internal protocol QRCodeViewDelegate: AnyObject {
+    
+    func saveAsImage(qrCodeImage: UIImage?, sourceView: UIView)
+
+    func copyToPasteboard(with action: QRCodeAction)
+}
+
+internal final class QRCodeView: UIView, Localizable, AdyenObserver {
+    
+    private let model: Model
+    
+    /// The delegate of the view
+    internal weak var delegate: QRCodeViewDelegate?
+    
+    public var localizationParameters: LocalizationParameters?
+    
+    internal init(model: Model) {
+        self.model = model
+        super.init(frame: .zero)
+        buildUI()
+        backgroundColor = model.style.backgroundColor
+        
+        accessibilityIdentifier = "adyen.QRCode"
+    }
+    
+    @available(*, unavailable)
+    internal required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func buildUI() {
+        addLogo()
+        addInstructionLabel()
+        addQRCodeImage()
+        addAmountToPay()
+        addProgressView()
+        addExpirationLabel()
+        if model.action.paymentMethodType == .pix {
```

Comment:
Hmm so some types will have copy code and some will have save as image. This decision better be in the viewmodel instead of the view. 
Let's add a property on the model that decides whether to show copyCode or saveImage (maybe enum?)